lib/v2/fda/router.js

```diff
@@ -1,3 +1,3 @@
 module.exports = function (router) {
-    router.get('/cdrh', require('./cdrh'));
+    router.get('/cdrh/:cate*', require('./cdrh'));
```

Comment:
```suggestion
    router.get('/cdrh/:cate?', require('./cdrh'));
```
Please also update the path in the docs and maintainer.js